.github/workflows/be-cd.yml

```
@@ -0,0 +1,60 @@
+          target: "~/app/backend/build/libs"
+
+      # 배포 스크립트 실행
+      - name: Deploy
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.EC2_HOST }}
+          username: ${{ secrets.EC2_USERNAME }}
+          key: ${{ secrets.EC2_SSH_KEY }}
+          script: |
+            cd ~/app
+            bash deploy_scripts/backend_deploy.sh
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)